modules/highlights.js

```diff
@@ -0,0 +1,31 @@
+const Augur = require('augurbot');
+const Module = new Augur.Module();
+Module.addCommand({name: "fetchhighlights", 
+    permissions: (msg) => ['337713155801350146'].includes(msg.author.id),
```

Comment:
All snowflakes should be kept in `snowflakes.json` with a matching entry in `snowflakes-example.json`.